src/sentry/static/sentry/app/views/discover/discover.tsx

```diff
@@ -471,12 +456,7 @@ export default class Discover extends React.Component<Props, State> {
 
         <DiscoverGlobalSelectionHeader
           organization={organization}
-          projects={currentQuery.projects}
           hasCustomRouting
-          relative={currentQuery.range}
-          start={start}
-          end={end}
-          utc={utc}
```

Comment:
These are not valid props: https://github.com/getsentry/sentry/blob/5743740b65d4d26a656cbb0f3481494ceead826d/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx#L48-L172


`<TimeRangeSelector>` would be the component to use these: https://github.com/getsentry/sentry/blob/5743740b65d4d26a656cbb0f3481494ceead826d/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx#L697-L710

But these values come from selection store:

https://github.com/getsentry/sentry/blob/5743740b65d4d26a656cbb0f3481494ceead826d/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx#L613